src/com/android/calendar/settings/AboutPreferences.kt

```diff
@@ -17,32 +17,61 @@
 
 package com.android.calendar.settings
 
-import android.content.pm.PackageManager.NameNotFoundException
+import android.content.Intent
+import android.content.pm.PackageManager
+import android.net.Uri
 import android.os.Bundle
-import androidx.preference.Preference
-import androidx.preference.PreferenceFragmentCompat
-import ws.xsoh.etar.R
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
 
-class AboutPreferences : PreferenceFragmentCompat() {
+import java.util.Calendar
+import ws.xsoh.etar.R
+import kotlinx.android.synthetic.main.about_preferences.*
 
-    override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
-        setPreferencesFromResource(R.xml.about_preferences, rootKey)
-        setVersion()
+class AboutPreferences : Fragment() {
```

Comment:
Because it's no longer a ``PreferenceFragmentCompat``, I would propose…
* moving it out of the settings package
* no longer calling it using the ``SettingsActivity``. Either put it in a new Activity or replace it inside the ``AllInOneActivity``

Reply:
Yeah, I think having it in a separate activity is fine for now. The ``AllInOneActivity`` should be refactored at some point following a modern architecture.